postgres/pgdump.go

```diff
@@ -461,6 +485,10 @@ func getTableName(conv *internal.Conv, n nodes.RangeVar) (string, error) {
 type constraint struct {
 	ct   nodes.ConstrType
 	cols []string
+	/* Fields used for FOREIGN KEY constraints: */
+	name       string
+	referCols  []string
+	referTable string
 }
 
 // extractConstraints traverses a list of nodes (expecting them to be
```

Comment:
The function comment is now stale.
Suggest rewriting to:

// extractConstraints traverses a list of nodes (expecting them to be
// Constraint nodes), and collects the constraints they represent.

Reply:
updated comment